js/meshopt_clusterizer.module.d.ts

```diff
@@ -0,0 +1,35 @@
+// This file is part of meshoptimizer library and is distributed under the terms of MIT License.
+// Copyright (C) 2016-2024, by Arseny Kapoulkine (arseny.kapoulkine@gmail.com)
+
+export class Bounds {
+	center: Float32Array;
+	radius: number;
+	coneApex: Float32Array;
+	coneAxis: Float32Array;
+	coneCutoff: number;
+	coneAxisS8: Int8Array;
+	coneCutoffS8: number;
+}
+
+export class MeshletBuffers {
+	meshlets: Uint32Array;
+	vertices: Uint32Array;
+	triangles: Uint8Array;
+	getMeshlet(index: number): Meshlet;
```

Comment:
It feels like it would be more consistent for this to be a module-scope function (MeshoptClusterizer.getMeshlet, or maybe extractMeshlet), and for Meshlet object to not have a reference to buffers.

Reply:
done: [894160e](https://github.com/zeux/meshoptimizer/pull/738/commits/894160e14564daded8e460ab580b22ea7034da27)

`buildMeshlets` now returns an object with the packed buffers and the meshlets array